django-rgd-3d/rgd_3d/tasks/etl.py

```diff
@@ -14,17 +14,19 @@
 logger = get_task_logger(__name__)
 
 
-def _file_conversion_helper(source, output_field, method, prefix='', extension='', **kwargs):
+def _file_conversion_helper(
+    source: ChecksumFile, output: ChecksumFile, method: callable, extension: str = '', **kwargs
```

Comment:
I'm not sure if `callable` is the proper type annotation here, [`typing.Callable`](https://docs.python.org/3/library/typing.html#callable) might be the correct one but idk. We don't run type checking on Girder 4 projects in general, so I guess this isn't a big deal if the code runs okay.

Reply:
Ah yeah, `callable` is a function, not a type. `typing.Callable` is definitely right, thanks